viz.yaml

```diff
@@ -41,5 +44,12 @@ process:
     depends: 
       waterUse: waterUse-data
 visualize:
+  -
+    id: wateruse-treemap
+    location: cache/treemaps/
+    visualizer: treemap
+    scripts: scripts/visualize/treemap.R
+    depends:
+      waterUse_clean: calc-waterData
```

Comment:
ok, thanks. not yet sure that additional `id`s solve the initial problems, but that's my bad for not getting more involved when you were discussing all this in recent PRs & issues. i think i can run with this for now